src/forces.f90

```diff
@@ -85,29 +89,35 @@ subroutine init_forces
        icvrt_lx(iv) = icvrt(iv)
        jcvlw_lx(iv) = max(jcvlw(iv)+1-xstart(2),1)
        jcvup_lx(iv) = min(jcvup(iv)+1-xstart(2),xsize(2))
+       kcvfr_lx(iv) = max(kcvfr(iv)+1-xstart(3),1)
+       kcvbk_lx(iv) = min(kcvbk(iv)+1-xstart(3),xsize(3))
        icvlf_ly(iv) = max(icvlf(iv)+1-ystart(1),1)
        icvrt_ly(iv) = min(icvrt(iv)+1-ystart(1),ysize(1))
        jcvlw_ly(iv) = jcvlw(iv)
        jcvup_ly(iv) = jcvup(iv)
+       kcvfr_ly(iv) = max(kcvfr(iv)+1-ystart(3),1)
+       kcvbk_ly(iv) = min(kcvbk(iv)+1-ystart(3),ysize(3))
     enddo
 
     if (nrank==0) then
        write(*,*) '========================Forces============================='
-       write(*,*) '                       (icvlf)      (icvrt) '
-       write(*,*) '                (jcvup) B____________C '
-       write(*,*) '                        \            \ '
-       write(*,*) '                        \     __     \ '
-       write(*,*) '                        \    \__\    \ '
-       write(*,*) '                        \            \ '
-       write(*,*) '                        \       CV   \ '
-       write(*,*) '                (jcvlw) A____________D '
+       write(*,*) '     (icvlf)      (icvrt)    (kcvbk)    (kcvfr)'
```

Comment:
Probably best to keep the old format when i2dsim = 1 and use the modified format only when i2dsim = 0 ?

Reply:
Got it. I will make the change.